ios/FluentUI/Label/Label.swift

```diff
@@ -17,22 +17,22 @@ public enum TextColorStyle: Int, CaseIterable {
     case warning
     case disabled
 
-    func color(for window: UIWindow) -> UIColor {
+    func color(fluentTheme: FluentTheme) -> UIColor {
         switch self {
         case .regular:
-            return Colors.textPrimary
+            return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground1])
         case .secondary:
-            return Colors.textSecondary
+            return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground3])
         case .white:
             return .white
         case .primary:
-            return Colors.primary(for: window)
+            return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.brandForeground1])
         case .error:
-            return Colors.error
+            return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.redForeground1])
```

Comment:
who uses label error, warning, disable style? do we still need to support them? especially error or warning? does this change impact badgefield or something?

Reply:
I'll dig into it and removed them if they're not used/needed anywhere.